core/utils/constants.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Elementor\Core\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Constants {
```

Comment:
```suggestion
abstract class Constants {
```
We don't want any instances of this class :)

Reply:
Perfect